<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetToolbox</class>
 <widget class="QWidget" name="TilesetToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gridSizeX">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gridSizeY">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="offsetX">
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="offsetY">
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="bootstrapButton">
     <property name="text">
      <string>Bootstrap</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
